### ISSUE TITLE
Add git submodule installation instructions to README.md

### ISSUE THREAD

**Issue by nate-double-u** [etcd-io/website#358]
## Issue

The `README.md` file's _Run the site locally: Setup_ section is missing a step.

## Solution

Add the submodule installation instructions before the `npm install` instruction.

`git submodule update --init --recursive --depth 1`

**Comment by nate-double-u** [comment]
Kubernetes has a similar requirement: https://github.com/kubernetes/website/#prerequisites

**Comment by kapunahelewong** [comment]
I can do this one! Thanks for the Kubernetes context too, @nate-double-u!

**Comment by chalin** [comment]
Btw, the npm commands should handle the submodules.

**Comment by chalin** [comment]
See the `get-submodules` npm command.

**Comment by nate-double-u** [comment]
> Btw, the npm commands should already handle the submodules.

This is correct, I'd forgotten the `get-submodules` npm command. I've updated the issue accordingly.
